README.md

```diff
@@ -289,6 +289,27 @@ Default Content (if template is unavailable):
 <p><a href="{{ .ConfirmationURL }}">Change Email</a></p>
 ```
 
+`WEBHOOK_URL` - `string`
+
+Url of the webhook listener server. This will called when events like  `validate`, `signup` or `login` occurs.
+
+`WEBHOOK_SECRET` - `string`
+
+Shared secret to authorize webhook requests.
```

Comment:
maybe add a note that this uses `JWS` signatures?

Reply:
I extended the description a little bit, also to explain why JWS verification is important.